CONTRIBUTING.md

```diff
@@ -9,9 +9,11 @@
 
 #### Coding Guidelines
 
-- All source code contributions must be formatted prior to checkin by running `terraform fmt`, and changes should not break validation checks provided by the `terraform validate` command.
+- All source code contributions must be formatted prior to checkin by running `terraform fmt -recursive`, and changes should not break validation checks provided by the `terraform validate` command.
```

Comment:
Should consider adding https://github.com/marketplace/actions/terraform-validate at a future point

Reply:
![image](https://github.com/NVIDIA/nvidia-terraform-modules/assets/37022534/15dd368b-a543-4036-aad2-39dafa36c5ca)
`terraform validate` is already part of this sentence. Is there a different way you want us to put it?

Reply:
No, I'm saying that the github action that automatically validates pull requests should be added to this repository at a future point

Reply:
I see, thanks for sharing the GitHub Actions link